.evergreen/run-kms-tls-tests.sh

```diff
@@ -10,11 +10,10 @@ set -o errexit  # Exit the script with error if any of the commands fail
 ############################################
 #            Main Program                  #
 ############################################
-
+RELATIVE_DIR_PATH="$(dirname "${BASH_SOURCE:-$0}")"
```

Comment:
Is `-PjavaVersion=${JAVA_VERSION}` needed in the gradle invocations below?

Reply:
No this has the same behaviour as before - JAVA_HOME is set to the default java 11.